### PR TITLE
Add debug components for easier testing.

### PR DESCRIPTION
**Pull request by awong-dev**
Adds PopulateVeteranButton button which prepopulates a bunch of fields
to make form testing way easier.

Adds a RoutesDropdown that allows navigating to subsections much faster.

### CODE REVIEW

File: src/client/components/HealthCareApp.jsx [hunk 1]
```diff
@@ -173,19 +179,36 @@ class HealthCareApp extends React.Component {
         </div>
       );
     }
+    let devPanel = undefined;
+    if (__DEV__) {
+      const queryParams = _.fromPairs(
+        window.location.search.substring(1).split('&').map((v) => { return v.split('='); }));
```
Comment (robbiethegeek) [inline src/client/components/HealthCareApp.jsx]: I have never done this, but it looks like react router provides the query params:
Here is an example from here : https://github.com/reactjs/react-router/blob/8707d05b00273590e4d054771b8ae7ae9ad312e3/examples/query-params/app.js
    let { query } = this.props.location
    let age = query && query.showAge ? '33' : ''

**Comment by crwallace** [comment]
@awong-dev this looks good but how do I run it with `dev` specified so that I can see that it works?

**Comment by crwallace** [comment]
Got it running and LGTM!

File: src/client/components/HealthCareApp.jsx (hunk 1 above)
Comment (awong-dev) [inline src/client/components/HealthCareApp.jsx]: Nice!  I was looking for something like that but couldn't find it.  I'll fix in a follow-up PR.

**Pull request merged**